pom.xml

```diff
@@ -179,13 +179,13 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId> <!-- Use 'netty-all' for 4.0 or above -->
-            <version>4.1.42.Final</version>
+            <version>4.1.43.Final</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-tcnative</artifactId>
-            <version>2.0.25.Final</version>
```

Comment:
How do you map the netty-all version to netty-tcnative ?

Reply:
I just picked the latest versions for both of them.